recipes/libhwloc/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+chmod +x configure
+
+DISABLES="--disable-cairo --disable-libxml2 --disable-opencl --disable-cuda --disable-nvml"
+DISABLES="$DISABLES --disable-gl --disable-libnuma --disable-libudev"
+
+if [ `uname` == Darwin ]; then
+    ./configure --prefix=$PREFIX --disable-shared $DISABLES
+else
+    ./configure --prefix=$PREFIX $DISABLES
+fi
+
+make
+#make check
```

Comment:
Is that failing? Can you add a comment about the failures? (We can still merge if you think the failures are not important.)